exampleCourse/questions/stringOperations/question.html

```diff
@@ -0,0 +1,66 @@
+<pl-question-panel>
+  <p> 1) For the following python code snippet, enter below the resulting string:</p>
+
+<pl-code language="python">
+a = "{{params.stringname}}"*{{params.a}}
+</pl-code>
+
+</pl-question-panel>
+
+<p>
+<pl-string-input answers_name="ans1"  label = "a = ">
+</pl-string-input>
+</p>
+
+<pl-question-panel>
+<p> 2) For the following python code snippet, what is d = b + c?</p>
+<pl-code language="python">
+  b = "{{params.b}}"
+  c = "{{params.c}}"
+</pl-code>
+</pl-question-panel>
+
+<p>
+<pl-string-input answers_name="ans2" label = "b + c = " >
+</pl-string-input>
+</p>
+
+<pl-question-panel>
+<p>3) Enter the string below, however include trailing and leading blanks.
+ When using the attribute <code>remove_leading_trailing=True</code>, the answer will
+ still be evaluated as correct. </p>
+<pl-code language="python">
+"{{params.d}}"
+</pl-code>
+</pl-question-panel>
+
+<p>
+<pl-string-input answers_name="ans3" remove_leading_trailing=True >
+</pl-string-input>
+</p>
+
+<pl-question-panel>
+<p> 4) Enter the string below, however include blank spaces between some of the numbers.
+When using the attribute <code>remove_between_spaces=True</code>, the answer will
+still be evaluated as correct. </p>
+<pl-code language="python">
+"{{params.d}}"
+</pl-code>
+</pl-question-panel>
+
+<p>
+<pl-string-input answers_name="ans4" remove_between_spaces=True >
+</pl-string-input>
+</p>
+
+
+<pl-question-panel>
+<p> 5) The attribute <code>allow_blank=True</code> allows users to submit blank answers, without
+  receiving the <code>INVALID</code> error. Leave the box below empty, and notice
+that you will not receive the <code>INVALID</code> feedback (and hence the question is graded). </p>
+</pl-question-panel>
+
+<p>
+<pl-string-input answers_name="ans5" label = "blank input box = " allow_blank=True >
```

Comment:
Remove the spaces on either side of the =; boolean attributes should be like `allow_blank="true"` (this might work now, but will probably fail on a future stricter parser.

Reply:
Fixed!